mlflow/server/js/src/experiment-tracking/components/CompareRunView.js

```diff
@@ -188,45 +265,87 @@ export class CompareRunView extends Component {
                       ? Utils.formatTimestamp(run.getStartTime())
                       : '(unknown)';
                     return (
-                      <td className='meta-info' key={run.run_uuid}>
-                        {startTime}
+                      <td className='data-value' key={run.run_uuid} style={colWidthStyle}>
+                        <Tooltip
+                          title={startTime}
+                          color='blue'
+                          overlayStyle={{ maxWidth: '400px' }}
+                          mouseEnterDelay={1.0}
+                        >
+                          {startTime}
+                        </Tooltip>
                       </td>
                     );
                   })}
                 </tr>
-                <tr>
-                  <th
-                    scope='rowgroup'
-                    className='inter-title'
-                    colSpan={this.props.runInfos.length + 1}
-                  >
-                    <h2>
-                      <FormattedMessage
-                        defaultMessage='Parameters'
-                        // eslint-disable-next-line max-len
-                        description='Row group title for parameters of runs on the experiment compare runs page'
-                      />
-                    </h2>
-                  </th>
-                </tr>
-                {this.renderDataRows(this.props.paramLists, true)}
-                <tr>
-                  <th
-                    scope='rowgroup'
-                    className='inter-title'
-                    colSpan={this.props.runInfos.length + 1}
-                  >
-                    <h2>
-                      <FormattedMessage
-                        defaultMessage='Metrics'
-                        // eslint-disable-next-line max-len
-                        description='Row group title for metrics of runs on the experiment compare runs page'
-                      />
-                    </h2>
-                  </th>
-                </tr>
+              </tbody>
+            </table>
+          </div>
+        </CollapsibleSection>
+        <CollapsibleSection
+          title={
+            <h1>
+              <FormattedMessage
+                defaultMessage='Parameters'
+                // eslint-disable-next-line max-len
+                description='Row group title for parameters of runs on the experiment compare runs page'
+              />
+            </h1>
+          }
+        >
+          <Switch
+            checkedChildren='Show diff only'
+            unCheckedChildren='Show diff only'
+            onChange={(checked, e) => this.setState({ onlyShowParamDiff: checked })}
+          />
+          <br />
+          <br />
+          <div className='responsive-table-container'>
```

Comment:
Do we need to wrap all the tables with this div?

Reply:
delete this div. But I don't delete `responsive-table-container`, this css class is used in other pages.